<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteMemoryWidget</class>
 <widget class="QWidget" name="WriteMemoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButton_writeMemory">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Ctrl + W</string>
          </property>
          <property name="text">
           <string>Write Memory</string>
          </property>
          <property name="icon">
           <iconset resource="../../icon.qrc">
            <normaloff>:/images/writeMemory.png</normaloff>:/images/writeMemory.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_stop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>F10</string>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="../../icon.qrc">
            <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>217</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Write Parameter</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>File Path:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>138</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Begin Address (HEX):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_region">
               <property name="text">
                <string>Region:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit_FilePath">
               <property name="minimumSize">
                <size>
                 <width>229</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_address">
               <property name="minimumSize">
                <size>
                 <width>229</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>0x1F600000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_region"/>
             </item>
            </layout>
           </item>
           <item row="0" column="2">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QToolButton" name="toolButton_openFile">
               <property name="minimumSize">
                <size>
                 <width>31</width>
                 <height>31</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Browser</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../icon.qrc">
                 <normaloff>:/images/openFolder.png</normaloff>:/images/openFolder.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_dram">
             <property name="text">
              <string>by DRAM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_sram">
             <property name="text">
              <string>by SRAM</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>484</width>
       <height>62</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icon.qrc"/>
 </resources>
 <connections/>
</ui>
